eingebaut.js

```diff
@@ -305,6 +305,12 @@ var Eingebaut = function(container, displayDevice, swfLocation, callback, option
     }
     return $this.video.prop('src')||'';
   };
+  $this.getPlaybackRate = function(){
+    return $this.video[0].playbackRate
```

Comment:
There could be cases where `playbackRate` isn't supported by the browser or devices; and we should guard against that with a default `1`.